lib/fluent/plugin/out_http.rb

```diff
@@ -21,6 +21,16 @@
 require 'fluent/plugin/output'
 require 'fluent/plugin_helper/socket'
 
+# patch Net::HTTP to support extra_chain_cert which was added in Ruby feature #9758.
+# see: https://github.com/ruby/ruby/commit/31af0dafba6d3769d2a39617c0dddedb97883712
+unless Net::HTTP::SSL_IVNAMES.include?(:@extra_chain_cert)
```

Comment:
@nurse I added your suggested fix here (changed `SSL_IVNAMES` to `Net::HTTP::SSL_IVNAMES`)
Would love your review on the PR.